parm/config/sfs/config.resources

```diff
@@ -0,0 +1,363 @@
+        ;;
+      *)
+        echo "FATAL ERROR: Resources not defined for job ${step} at resolution ${CASE}"
+        exit 4
+        ;;
+    esac
+
+    unset NTASKS_TOT
+    ;;
+
+  "atmos_products")
+    # Walltime is per forecast hour; will be multipled by group size
+    export walltime="00:15:00"
+    export ntasks=24
+    export threads_per_task=1
+    export tasks_per_node="${ntasks}"
+    export is_exclusive=True
+    ;;
+
+  "atmos_ensstat")
+    # Walltime is per forecast hour; will be multipled by group size
+    export walltime="00:15:00"
+    export ntasks=6
+    export threads_per_task=1
+    export tasks_per_node="${ntasks}"
+    export is_exclusive=True
+    ;;
+
+  "oceanice_products")
+    # Walltime is per forecast hour; will be multipled by group size
+    export walltime="00:15:00"
+    export ntasks=1
+    export tasks_per_node=1
+    export threads_per_task=1
+    export memory="96GB"
+    ;;
+
+  "wavepostsbs")
+    # Walltime is per forecast hour; will be multipled by group size
+    export walltime="00:15:00"
+    export ntasks=1
+    export threads_per_task=1
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export NTASKS=${ntasks}
+    export memory="10GB"
+    ;;
+
+  # The wavepost*pnt* jobs are I/O heavy and do not scale well to large nodes.
+  # Limit the number of tasks/node to 40.
+  "wavepostbndpnt")
+    export walltime="03:00:00"
+    export ntasks=240
+    export threads_per_task=1
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    if [[ ${tasks_per_node} -gt 40 ]]; then
+        export tasks_per_node=40
+        export is_exclusive=False
+    fi
+    export NTASKS=${ntasks}
+    ;;
+
+  "wavepostbndpntbll")
+    export walltime="01:00:00"
+    export ntasks=448
+    export threads_per_task=1
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    if [[ ${tasks_per_node} -gt 40 ]]; then
+        export tasks_per_node=40
+        export is_exclusive=False
+    fi
+    export NTASKS=${ntasks}
+    ;;
+
+  "wavepostpnt")
+    export walltime="04:00:00"
+    export ntasks=200
+    export threads_per_task=1
+    export tasks_per_node=$(( max_tasks_per_node / threads_per_task ))
+    export is_exclusive=True
+    if [[ ${tasks_per_node} -gt 40 ]]; then
+        export tasks_per_node=40
+        export is_exclusive=False
+    fi
+    export NTASKS=${ntasks}
+    ;;
+
+  "extractvars")
+    export walltime="00:30:00"
+    export ntasks=1
+    export threads_per_task=1
+    export tasks_per_node="${ntasks}"
+    export walltime_gfs="${walltime}"
+    export ntasks_gfs="${ntasks}"
+    export threads_per_tasks_gfs="${threads_per_task}"
+    export tasks_per_node_gfs="${tasks_per_node}"
+    export is_exclusive=False
+    ;;
+
+  "arch")
+    export walltime="06:00:00"
+    export ntasks=1
+    export tasks_per_node=1
+    export threads_per_task=1
+    export memory="4096M"
+    ;;
```

Comment:
Should have two jobs sets of resources here.  One for `arch_tars` and one for `arch_vrfy`.